build.gradle

```diff
@@ -1,15 +1,21 @@
+import org.gradle.api.file.DuplicatesStrategy
+
 /*
  * Gradle build file for JGribX
  *
- * Target Gradle version: 7.3.2
+ * Target Gradle version: 7.3.3
  *
  * Reference: https://docs.gradle.org/7.3/userguide/building_java_projects.html
  *
  */
+plugins {
+    id "java"
+    id "maven-publish"
+    id "eclipse"
+}
 
-// Apply the java plugin to add support for Java
-apply plugin: 'java'
-apply plugin: 'eclipse'
+version = 0.1
```

Comment:
Any reason not to line this up with the original version? Looks like 0.5.0 was the last released, maybe we go with `0.6.0-SNAPSHOT` here?

Reply:
I was going along a "CTI Version" so I will revert, re-upload, and update MrHab as well.